src/views/Dashboard/Customer/Home/MyTest/index.vue

```diff
@@ -137,35 +179,187 @@ export default {
         createdAt: new Date(parseInt(result._source.createdAt)).toLocaleDateString(),
         timestamp: parseInt(result._source.createdAt)
       }))
-      this.orderHistory = this.orderHistory.filter(order => order.status == "OrderPaid")
     },
 
-    setStatusColor(status) {
+    setStatusColor(status) { //change color for each order status
       let colors = Object.freeze({
-        REGISTERED: "#44921A",
-        PAID: "#E27625",
-        RESULTREADY: "#5640A5",
+        REGISTERED: "#6F4CEC",
+        PAID: "#44921A",
+        UNPAID: "#FAAD15",
+        FULFILLED: "#44921A",
+        CANCELLED: "#9B1B37",
+        FAILED: "#9B1B37",
+        RESULTREADY: "#6F4CEC",
         REJECTED: "#9B1B37",
         SUBMITEDASDATABOUNTY: "#5640A5"
       })
-      return { color: colors[status.toUpperCase()] }
+      return colors[status.toUpperCase()]
+    },
+
+    async getOrderHistory() {//this for get order from substrate
+      console.log("masuk ke get order history")//check mounted function
```

Comment:
please remove all the console.log which is not related to the code itself

Reply:
all unnecessary console.log removed